protocols/delegate-manager/contracts/DelegateManager.sol

```diff
@@ -0,0 +1,164 @@
+/*
+  Copyright 2019 Swap Holdings Ltd.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+pragma solidity 0.5.10;
+pragma experimental ABIEncoderV2;
+
+import "@airswap/indexer/contracts/interfaces/IIndexer.sol";
+import "@airswap/swap/contracts/interfaces/ISwap.sol";
+import "@airswap/delegate-factory/contracts/interfaces/IDelegateFactory.sol";
+import "@airswap/delegate/contracts/interfaces/IDelegate.sol";
+import "@airswap/types/contracts/Types.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+
+contract DelegateManager is Ownable {
+
+    event DelegateCreated(address owner, address delegate);
+
+    //keeps track of all the delegates created by owner address
+    mapping(address => address[]) private _ownerToDelegates;
+
+    //keeps track of the staked amounts per msg.sender
+    //signer => sender => msg.sender => staked amount
+    mapping(
+      address => mapping(
+        address => mapping(
+          address => uint256
+        )
+      )
+    ) public stakedAmounts;
```

Comment:
what's the motivation for storing this mapping in the DelegateManager? The information is all stored in the Indexers, so was just wondering why it's being stored here too?

Reply:
Good question.

I think I've confused myself here. 
I tripped up on Manager being owned or Manager acting on behalf of many users.

Reply:
A Delegate manager is now owned and does not interface with multiple parties.